CHANGELOG.md

```diff
@@ -1,9 +1,24 @@
-## 3.1.5 (TBD, 2024)
+# 3.1.5 (July 18, 2024)
+
 ###
-- Supported Controller version: **UserConnect-TBD**
+- Supported Controller version: **UserConnect-7.1.4105**
 - Supported Terraform version: **v1.x**
 
-### Deprecations
+## Bug Fixes:
```

Comment:
Due to the missed "#" in earlier headings, it seems to have waterfalled all the other headings to be missing that character
```suggestion
### Bug Fixes:
```